bitbots_localization/include/bitbots_localization/MotionModel.h

```diff
@@ -0,0 +1,69 @@
+//
+// Created by judith on 09.03.19.
+//
+
+#ifndef BITBOTS_LOCALIZATION_MOTIONMODEL_H
+#define BITBOTS_LOCALIZATION_MOTIONMODEL_H
+
+#include <memory>
+#include <stdlib.h>
+
+#include <particle_filter/MovementModel.h>
+#include <bitbots_localization/RobotState.h>
+#include <geometry_msgs/Vector3.h>
+#include <particle_filter/CRandomNumberGenerator.h>
+#include <bitbots_localization/tools.h>
+
+/**
+ * @class MyMovementModel
+ *
+ * @brief Test class for ParticleFilter.
+ *
+ * This simple movement model only adds a small jitter in diffuse() and does
+ * nothing in drift().
+ * @author Stephan Wirth
+ */
+class RobotMotionModel : public particle_filter::MovementModel<RobotState> {
+
+public:
+    /**
+     * empty
+
+     */
+    RobotMotionModel(particle_filter::CRandomNumberGenerator& random_number_generator, double xStdDev, double yStdDev, double tStdDev, double multiplicator);
+
+    /**
+     * empty
+     */
+    ~RobotMotionModel();
```

Comment:
If you define a non default destructor please also define the others as well: a copy constructor, a copy assignment operator, a move constructor or a move assignment operator.

Exsampe by ISOCPP:
```
X(const X&) = default;                         // copy constructor
    X& operator=(const X&) = default;  // copy assignment
    X(X&&) = default;                            // move constructor
```
or
```
    X& operator=(X&&) = default;         // move assignment
```

Reply:
Would `~RobotMotionModel() = default;` work instead?